libs/infra-nest-server/src/lib/processJob.ts

```diff
@@ -5,7 +5,7 @@ export const processJob = () =>
   yargs(hideBin(process.argv))
     .option('job', {
       string: true,
-      choices: ['worker', 'server', 'cleanup'] as const,
+      choices: ['worker', 'server', 'cleanup', 'birthday'] as const,
```

Comment:
This should not be needed. It should just be a worker type, this change likely has unintended affects which will break CI


```suggestion
      choices: ['worker', 'server', 'cleanup'] as const,
```

Reply:
Ah, makes sense now. Thanks.